### PR TITLE
add rule for Volantix.com:

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

**Comment by J0WI** [comment]
Does this site have any content?

File: src/chrome/content/rules/Volantix.com.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!--
+		Other Instart Logic rulesets:
+
+		- Insnw.net.xml
+		- InstartLogic.com.xml
```
Comment (J0WI) [inline src/chrome/content/rules/Volantix.com.xml]: Please also update the reference list there.

**Comment by numismatika** [comment]
It is just referenced by other pages as third party object. Suspect it just serves as tracking host.

**Comment by J0WI** [comment]
@numismatika If they are related the references should be updated on those other files. If they are not related, you can remove the reference list from this rule.

**Pull request merged**